modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMFormFieldFactoryHelper.java

```diff
@@ -221,6 +221,10 @@ protected LocalizedValue getDDMFormFieldPredefinedValue() {
 		String fieldType = getDDMFormFieldType();
 
 		if (Validator.isNotNull(predefinedValue)) {
+			if (StringUtil.startsWith(predefinedValue, StringPool.PERCENT)) {
+				return createLocalizedValue(predefinedValue);
+			}
```

Comment:
@carolmariaabb, can you please give more details about this code? Why is it looking for the percent sign?

Reply:
Hi @natocesarrego, sure!

For example, in this [commit](https://github.com/natocesarrego/liferay-portal/pull/980/commits/dd165707a88d9ccf5715f8753e8d32252ac78eaf), I set the predefined value for the `confirmationErrorMessage` to `%the-information-does-not-match`. Therefore, this code checks whether the predefined value contains a language key to be translated. 
Before this code, it was not possible to set a language key for the predefined value, it only worked for the `label` property.

Reply:
Great! Thanks for the explanation. :wink: